storage/storageUnit/nilStorer.go

```diff
@@ -16,10 +16,18 @@ func (ns *nilStorer) Get(key []byte) ([]byte, error) {
 	return nil, nil
 }
 
+func (ns *nilStorer) GetFromEpoch(key []byte, epoch uint32) ([]byte, error) {
```

Comment:
Why do we have a nilStorer? I would say that no component should start without a storer or with a nilStorer. Could we remove, maybe in another PR the nilStorer?

Reply:
it was needed for the `Null object design pattern` because not all the components were available when creating the status handler persister. Fwd to @miiu96

Reply:
ok! I will also take a look on that